CAP/gap/UniversalObjects.gi

```diff
@@ -503,6 +503,74 @@ InstallGlobalFunction( FiberProduct,
     
 end );
 
+##
+InstallOtherMethod( FiberProductEmbeddingInDirectProduct,
+        [ IsList ],
+        
+  function( diagram )
+    
+    local sources_of_diagram, cat, test_source;
+    
+    sources_of_diagram := List( diagram, Source );
+    
+    cat := CapCategory( diagram[1] );
```

Comment:
This should simply extract the category and then call `FiberProductEmbeddingInDirectProduct` with two arguments.